app/views/spree/admin/permission_sets/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="alert alert-info"><%= Spree.t(:permission_set_admin_permission_required) %></div>
+
+<div data-hook="admin_permission_set_form_fields">
+  <%= f.field_container :name, class: ['form-group'] do %>
+    <%= f.label :name, Spree.t(:permission_set_name) %><span class="required"></span>
+    <%= f.text_field :name, class: 'form-control' %>
+    <%= f.error_message_on :name, class: 'error-message' %>
+  <% end %>
+  <%= f.field_container :description, class: ['form-group'] do %>
+    <%= f.label :description, Spree.t(:permission_set_description) %>
+    <%= f.text_field :description, class: 'form-control' %>
+    <%= f.error_message_on :description, class: 'error-message' %>
+  <% end %>
+  <%= f.field_container :display_permission, class: ['form-group'] do %>
+    <%= f.label :display_permission, Spree.t(:permission_set_display_permission) %>
+    <p class="help-block"><%= Spree.t(:permission_set_display_permission_help) %></p>
+    <%= f.check_box :display_permission, class: 'form-control' %>
```

Comment:
@nimish-mehta Checkbox design is broken, I think you need to remove form-control class from checkbox field.